Models/Article.js

```diff
@@ -0,0 +1,13 @@
+export default class Article {
+  constructor(title, content, writer, likeCount = 0, createdAt = new Date()) {

```

Comment:
- likeCount 같은 경우 보통 인스턴스가 만들어졌을 때 0의 값을 갖으니 매개변수로 받지 말고 그냥 바로 0 값을 할당하는 것도 좋을 것 같습니다.

```suggestion
  constructor(title, content, writer, createdAt = new Date()) {

```